components/prombench/manifests/benchmark/3_prometheus-test.yaml

```diff
@@ -747,12 +758,22 @@ spec:
           initialDelaySeconds: 30
           periodSeconds: 2
           failureThreshold: 30
-        args: ["{{ .PR_NUMBER }}"]
+        command: ["/usr/bin/prometheus"]
+        args: [
+          "--web.external-url=http://prombench.prometheus.io/{{ .PR_NUMBER }}/prometheus-pr",
+          "--storage.tsdb.path=/prometheus",
+          "--web.console.libraries=/usr/bin/console_libraries",
+          "--web.console.templates=/usr/bin/consoles",
+          "--config.file=/etc/prometheus/prometheus.yml",
```

Comment:
btw why do we need all these additional flags? Aren't some of these added by default? 

`--log.level=debug` is good to keep

Reply:
we can get rid of the `--web.console.*` flags as the point to the executable's running directory.

Also can we remove the Makefiles inside `/temp/prometheus-builder/` ? I don't see them being used.

Reply:
> we can get rid of the `--web.console.*` flags as the point to the executable's running directory. I'll add it

Thanks, if we can get rid of anything else please remove it. 




> Also can we remove the Makefiles inside `/temp/prometheus-builder/` ? I don't see them being used.

yes I  noticed this today as well.

Reply:
> we can get rid of the --web.console.* flags as the point to the executable's running directory.

I can see that for the other deployment it is still there. Do you know why are these needed?

Reply:
from [k8s docs](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/):
> If you supply only args for a Container, the default Entrypoint defined in the Docker image is run with the args that you supplied.

While the release docker image does put most of the flags at the same [places](https://github.com/prometheus/prometheus/blob/master/Dockerfile), it does not have have the `--web.external.url` flag so have to have to specify the other flags that are different from default with it.

the [Dockerfile](https://github.com/prometheus/prometheus/blob/master/Dockerfile) is `COPY`ing console files to `/usr/share/prometheus/console_libraries/` and then softlinking it to `/etc/prometheus` so our args in release deployment object are working and required.

Reply:
in that case lets try to keep the one we build and the one we run from the release images as identical as possible for easy maintenance.